src/resolvers/Query/eventsByUserType.ts

```diff
@@ -0,0 +1,90 @@
+import type { QueryResolvers } from "../../types/generatedGraphQLTypes";
+import type { InterfaceEvent, InterfaceUserAttende } from "../../models";
+import { Event, Organization, User } from "../../models";
+import { STATUS_ACTIVE } from "../../constants";
+import { getSort } from "./helperFunctions/getSort";
+/**
+ * This query will fetch all events for the organization which have `ACTIVE` status from database.
+ * @param _parent-
+ * @param args - An object that contains `orderBy` to sort the object as specified and `id` of the Organization.
+ * @returns An `events` object that holds all events with `ACTIVE` status for the Organization.
+ */
+
+export const eventsByUserType: QueryResolvers["eventsByUserType"] = async (
+  _parent,
+  args
+) => {
+  const sort = getSort(args.orderBy);
+
+  const user = await User.findById({ _id: args.userId });

```

Comment:
Make use of either `findOne` or `find` mongoose functions, `findById` is very niche and specific to `id` which is not really required. This is to ensure there's a certain level of consistency in using mongoose functions.